src/pages/forms/createAccount.js

```diff
@@ -42,18 +42,35 @@ const CreateAccount = () => {
   const [formErrors, setFormErrors] = useState({});
 
   const onChangeHandler = (e) => {
-    console.log(e.target.value);
     const { name, value } = e.target;
     setFormValues({ ...formValues, [name]: value });
   };
 
-  const submitHandler = (e) => {
+  const submitHandler = async (e) => {
     e.preventDefault();
     setFormErrors(validate(formValues));
-  };
 
+    if (
+      errors.passwordIndicator === false &&
+      errors.emailIndicator === false &&
+      errors.firstNameIndicator === false &&
+      errors.lastNameIndicator === false &&
+      errors.businessNameIndicator === false
+    ) {
+      const response = await fetch(
+        "http://15.206.121.23:7077/v1/auth/email/registration/brand",
```

Comment:
this base url should be stored at a common place like environment.ts file 
baseUrl = http://15.206.121.23:7077/v1
and should be fetched from that file like 
await fetch(
        {baseUrl}/auth/email/registration/brand",

Reply:
this is done in order to avoid writing base url in every api integration and it is best practice also .
chcek docs how to fetch api url from common file